src/components/atoms/BoxSimple/BoxSimple.scss

```diff
@@ -1,30 +1,44 @@
 @import "colors";
 
 .BoxSimple {
-  padding: 0px 18px;
+  padding: 0rem 1.125rem;
+
+  &__image {
+    margin-left: 50%;
+    max-width: 8.75rem;
+    transform: translateX(-50%);
+    @media screen and (min-width: 48rem) {
```

Comment:
is there any reason you didn't create a new file with this breakpoints?

Reply:
I understood Lautaro's comment about the Nested vs Grouped media queries to mean that we would like to keep them nested in the same file. I am fine with changing this if that is what you would like.

Reply:
Oh, yes it's right media queries inside components, I wanted to mean, why didn't move the size `48em` move to a specific file.

Reply:
Well we agreed to avoid fixed breakpoints, but I see that we're kind of using them anyways, just not declaring them as constants. I see

Reply:
I think the fact that these components share ( can use ) the same breakpoint is more of a coincidence than a rule. But I am open to trying named (fixed) breakpoints if we want to try it.

Here are the material ui breakpoints, slightly adjusted, just throwing these out as a starting point, if someone has some they would like to recommend feel free.

```
default:     0px or larger
small:       600px (37.5rem) or larger
medium:      960px (60rem) or larger
large:       1280px (80rem) or larger
xlarge:      1920px (120rem) or larger
```

Reply:
@wzalazar if you have used named(fixed) breakpoints in the past can you give us some of the rules you used to make it successful? I have attempted them many times in the past but they always made the project more difficult to maintain in the long run. I found myself ditching them as requirements and content changed over time.

Reply:
To clarify a bit more, I think my experience may differ because of my previous jobs, I worked with designers that would design components without any consideration for current breakpoints and such, so it was impossible to fit the things they designed in the same breakpoints. It is quite possible we could make named ( fixed ) breakpoints work for us. I think the main difference is that once we have set named (fixed) breakpoints we will then design components around those breakpoints instead of the other way around.  :)

Reply:
I understand your point of view @kennylavender and agree because we don't the full control over the design. Another point is we decide to use rem for use breakpoint and in this case, the name is an anecdote because it will depend on our users if one user has another size in your browser the breakpoint will change.
But by another side, we need to avoid has repetitions of the same values.

Reply:
@wzalazar I am a little unsure as to what you're saying, but I think I understand. Let me know if I am correct here or not.

I think we are in agreement not to have `global` pre-defined breakpoints, but each component should define its own breakpoint variables so that the breakpoint values are not repeated through its sass file.

If that is what you are saying I am onboard with that :)

Reply:
Since we don't know yet, I'd default to assuming our designer won't design things in compatibility with fixed breakpoints.

> I think the fact that these components share ( can use ) the same breakpoint is more of a coincidence than a rule.
Agree! 


> But by another side, we need to avoid have repetitions of the same values.
Agree too.

For now, I think the best approach (so we don't dedicate a lot of time re-checking each breakpoint) would be to leave them as is, but if there's a reason for shared values, leaving them as declared constants _inside_ the component (so more than one component can't share the same break point, but the component is free to use it as many times as it wants inside).

Reply:
@lautarodragan yeah, is the better approach